easybuild/easyconfigs/p/psutil/psutil-5.9.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'PythonPackage'
+
+name = 'psutil'
+version = '5.9.0'
```

Comment:
There is `psutil` 5.8.0 included in `Python-3.9.6-GCCcore-11.2.0.eb`. That would be the usual route to get `psutil`, unless that version is unsuitable for what you are using it for.

Reply:
Thanks @branfosj, that's useful to know.  I'll ditch this pull request - is there a process I should follow for that?  

Just a question about this, for my future reference:

I tried to base this new version on the previous latest easyconfig for psutil, which contained:

```
multi_deps = {'Python': ['3.7.2', '2.7.15']}
```

I wanted to move both specified Python versions along to the latest ones with the latest toolchains, but there were only the "bare" builds available: Python-2.7.18-GCCcore-11.2.0-bare.eb and Python-3.9.6-GCCcore-11.2.0-bare.eb.  I could not find a way to get the multi_deps parameter to pick up these versions.  Does its syntax allow for this?  (No worries if not - as I say I'll not pursue this PR.)

Reply:
@Ghepardo to close this pull request, just push the "Close pull request" button at the bottom and add a comment why you're closing it.

I think there is currently no way to specify the -bare modules with multi-deps. In any case, the -bare modules should normally only be used as build dependencies.